db/migrate/20190820095253_copy_enrichments_to_next_cycle.rb

```diff
@@ -0,0 +1,20 @@
+class CopyEnrichmentsToNextCycle < ActiveRecord::Migration[5.2]
+  def up
+    current_providers = RecruitmentCycle.current_recruitment_cycle.providers.includes(:enrichments).all
+    next_providers = RecruitmentCycle.next_recruitment_cycle.providers.all
+
+    current_providers.each do |p|
+      next if p.enrichments.empty?
```

Comment:
`current_providers` should we do the filtering beforehand?
ie `current_providers_with_enrichments`?